src/highlevel_types.jl

```diff
@@ -19,7 +19,7 @@ end
 #Sets a field of JUA_XXX object to a JUA_YYY object, calls the next method, i.e., 
 #will create a copy of the JUA_YYY object. 
 function Base.setproperty!(x::AbstractOpen62541Wrapper, f::Symbol, v::AbstractOpen62541Wrapper)
-    @warn "Attention! Assigning a $(typeof(v)) as content of field $(String(f)) in a $(typeof(x)) leads to a copy of 
```

Comment:
I changed this because the @warn causes the resulting message to start with "Warning: ..." anyway. Unfortunately, this seems to make Codecov unhappy again.

Reply:
I think your version is better. No need to have a double warning :)